alpha/scripts/utils/getZoomToken.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+if($argc != 2)
+{
+	echo "Arguments missing.\n\n";
+	echo "Usage: php getZoomToken.php {vendor account id}\n";
```

Comment:
When you want to specify fields in the usage you should use < description > and not { }